src/styles/preset.css

```diff
@@ -125,6 +126,81 @@
   --color-severity-med: var(--oui-severity-med);
   --color-severity-high: var(--oui-severity-high);
   --color-severity-critical: var(--oui-severity-critical);
+
+  /* Border Radius */
+  --radius-sm: calc(var(--oui-radius) - 4px);
+  --radius-md: calc(var(--oui-radius) - 2px);
+  --radius-lg: var(--oui-radius);
+  --radius-xl: calc(var(--oui-radius) + 4px);
+
+  /* Font Family */
+  --font-sans: Rubik, ui-sans-serif, system-ui, sans-serif;
+  --font-mono: 'Source Code Pro', ui-monospace, monospace;
+
+  /* Font Sizes with line heights */
+  --text-xs: 12px;                /* 12px */
+  --text-xs--line-height: 16px;   /* 16px */
+  --text-sm: 14px;                /* 14px */
+  --text-sm--line-height: 20px;   /* 20px */
+  --text-base: 14px;              /* 16px */
+  --text-base--line-height: 20px; /* 24px */
+  --text-lg: 18px;                /* 18px */
+  --text-lg--line-height: 28px;   /* 28px */
+  --text-xl: 20px;                /* 20px */
+  --text-xl--line-height: 28px;   /* 28px */
+  --text-2xl: 24px;               /* 24px */
+  --text-2xl--line-height: 32px;  /* 32px */
+  --text-3xl: 30px;
+  --text-3xl--line-height: 36px;
+  --text-4xl: 36px;
+  --text-4xl--line-height: 40px;
+  --text-5xl: 48px;
+  --text-5xl--line-height: 1;
+  --text-6xl: 60px;
+  --text-6xl--line-height: 1;
+  --text-7xl: 72px;
+  --text-7xl--line-height: 1;
+  --text-8xl: 96px;
+  --text-8xl--line-height: 1;
+  --text-9xl: 128px;
+  --text-9xl--line-height: 1;
+
+  /* Spacing Scale */
+  --spacing-0: 0;              /* 0 */
+  --spacing-px: 1px;           /* 1 */
+  --spacing-0\.5: 2px;         /* 2 */
```

Comment:
are we intentionally explicitly defining all of these? if the existing calculations work, can we just change the base?